discord-groups/style.css

```diff
@@ -61,21 +70,31 @@
 .groups {
   overflow: scroll;
 }
+.groups::-webkit-scrollbar {
+  width: 5.5px;
+  height: 5.5px;
+}
+.groups::-webkit-scrollbar-thumb {
+  background: var(--color-buttons-background);
+  border-radius: 20px;
+}
 .groups-list {
   padding: 1rem;
   list-style: none;
-  overflow: scroll;
-  height: 80vh;
+  /* overflow: scroll; */
+  height: 75vh;
 }
```

Comment:
remove comments

Reply:
why did you change it to 75 and not 80?

Reply:
removed comments.
I thought it will overflow. I changed it back to 80.